addons/stock_account/models/product.py

```diff
@@ -605,6 +605,9 @@ def _compute_average_price(self, qty_invoiced, qty_to_invoice, stock_moves):
         """
         self.ensure_one()
 
+        if not qty_to_invoice:
+            return 0
```

Comment:
what?

Reply:
Without this check, we can have a division by zero in line "return tmp_value / qty_to_invoice". (traceback: https://pastebin.com/W9k120Hy)

Reply:
according to your traceback, this call triggers the issue
``` python
line.purchase_price = line.product_id._compute_average_price(0, line.product_uom_qty, line.move_ids)
```
so qty_to_invoice is line.product_uom_qty and this field cannot be equals to zero
``` python
product_uom_qty = fields.Float(string='Quantity', digits='Product Unit of Measure', required=True, default=1.0)
```

how can i reproduce this traceback in master ?

Reply:
Here is a video to reproduce this traceback in master https://drive.google.com/open?id=1nLt4sBGGaZuzjP3f8bAJogfTcpfvIZ5N

Reply:
it seems float required are useless.. so fine by me